website/docs: add webhook mapping examples (#21971)
goauthentik/authentik: website/docs/sys-mgmt/events/transports.md

Reviewer: ```suggestion
##### Webhook body examples
```

Reviewer: We could maybe add titles to the codeblocks but not essential

}
```

You can also include fields from the notification recipient and the triggering event:

```python
return {
    "email": request.user.email,
    "client_ip": notification.event.client_ip,
}
```

> For failed login notifications, the attempted username is stored on the event context. If the GeoIP and ASN context processors are configured, their data is also available from the same context:

Reviewer: ```suggestion
For failed login notifications, the attempted username is stored in the event context. If the GeoIP and ASN context processors are configured, their data is also available in the event context:
```
